src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,14 +37,18 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
   private final MecanumDriveSubsystem m_Drive = new MecanumDriveSubsystem();
 
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  //private final CommandXboxController m_driverController =  new CommandXboxController(OperatorConstants.kDriverControllerPort);
+  private final LanuchMechanisumSubsystem m_LaunchMech = new LanuchMechanisumSubsystem();
+  private final IntakeMechanisum m_Intakemech = new IntakeMechanisum();
+  private final TrapScoreSubsystem m_TrapScoreSubsystem = new TrapScoreSubsystem();
+
+  private Map<String, RelativeEncoder> Encoders = m_Drive.GetEncoders();
+
+  private final LimelightNavigation m_Navigation = new LimelightNavigation(Encoders);
```

Comment:
I think you can leave limelight navigation as its own subsystem, but it _may_ be better or cleaner to just include the navigation components all as part of the Drive Subsystem - if the other subsystems are taking actions based off the navigation, then it's just a matter of writing triggers.

Reply:
Moved into #26